config/httpd-modules.conf

```diff
@@ -5,3 +5,17 @@ LoadModule dir_module modules/mod_dir.so
 LoadModule authz_core_module modules/mod_authz_core.so
 LoadModule unixd_module modules/mod_unixd.so
 LoadModule mpm_event_module modules/mod_mpm_event.so
+LoadModule wsgi_module modules/mod_wsgi_python3.so
+LoadModule ssl_module modules/mod_ssl.so
+LoadModule env_module modules/mod_env.so
+LoadModule proxy_module modules/mod_proxy.so
+LoadModule proxy_ajp_module modules/mod_proxy_ajp.so
+LoadModule proxy_balancer_module modules/mod_proxy_balancer.so
+LoadModule proxy_http_module modules/mod_proxy_http.so
+LoadModule slotmem_shm_module modules/mod_slotmem_shm.so
+LoadModule headers_module modules/mod_headers.so
+LoadModule authn_core_module modules/mod_authn_core.so
+LoadModule auth_basic_module modules/mod_auth_basic.so
+LoadModule authn_file_module modules/mod_authn_file.so
+LoadModule authz_user_module modules/mod_authz_user.so
```

Comment:
Does Metal3 use the httpd for anything these days other than Ironic with this PR? Sorry I haven't followed closely lately. But, in OpenShift we do use it for an image cache, and don't need any of these modules. It's a lot of extraneous modules to enable.

Reply:
The modules are needed for:
- WSGI
- SSL handling: We make Apache server handle TLS part of the connection.
- Basic authentication: Apache server also handle basic auth.
That is the reason why we need that much modules.